ru_RU/webapp/src/main/webapp/templates/freemarker/userAccounts-passwordResetPendingEmail_ru_RU.ftl

```diff
@@ -0,0 +1,49 @@
+<#-- $This file is distributed under the terms of the license in /doc/license.txt$ -->
+
+<#-- Confirmation email for user account password reset -->
+
+<#assign subject = "Запрос на изменение пароля для сайта ${siteName}" />
+
+<#assign html>
+<html>
+    <head>
+        <title>${subject}</title>
+    </head>
+    <body>
+        <p>Уважаемый пользователь ${userAccount.firstName} ${userAccount.lastName}:</p>
+        
+        <p>Мы получили запрос на сброс пароля для вашей учётной записи на сайте ${siteName}, связанной с адресом (${userAccount.emailAddress}).</p>
+        
+        <p>Пожалуйста, следуйте приведенным ниже инструкциям, чтобы выполнить сброс пароля.</p>
+        
+        <p>Если вы не запрашивали новую учетную запись, можете смело игнорировать это письмо. Срок действия этого запроса истечет, если в течение 30 дней не будет предпринято никаких действий.</p>
+        
+        <p>Щёлкните по ссылке ниже или вставьте её в адресную строку браузера, чтобы сбросить пароль с помощью нашего защищенного сервера.</p>
+        
+        <p>${passwordLink}</p>
```

Comment:
In userAccounts-acctCreatedEmail ftl file, there is a link 

\<a href="${passwordLink}" title="password"\>${passwordLink}\</a\>

Here is just a text <p>${passwordLink}</p>, i.e. the user has to copy it to the address line of the web browser manually.

Reply:
Fixed